docs/features/transformers.md

```diff
@@ -56,16 +56,24 @@ Pulls a field out of a complex value and replaces the entire value with the extr
       field: id
 ```
 
-## Split Value
-Split an array into values.
+## Split array
+Split an array into multiple records.
 
 ```yaml
     - name: splitFooArray
       type: dev.vox.platform.kahpp.configuration.transform.SplitValueTransform
       config:
         jmesPath: "value.fooArray"
+        to: "value.foo" # Optional field
 ```
 
+### Optional configurations
+
+| name | default | description                              |
+|------|---------|------------------------------------------|
+| to   | ""      | define where to put the object extracted |
```

Comment:
this is the explanation @vadzim-kazak 
Is it not clear?

Reply:
yep, for me without KaHPP knowledge it is not clear. I understand this transformer as for any object with a nested array we are going to split this array to separate records. Does it mean that if "to" property is not empty a new object is going to be created and array value stored in it under "to" path? If yes, I guess we can describe this behavior under description column.

Reply:
Good idea!

Reply:
Updated